hack/toolchain.sh

```diff
@@ -2,7 +2,7 @@
 set -euo pipefail
 
 K8S_VERSION="${K8S_VERSION:="1.27.x"}"
-KUBEBUILDER_ASSETS="${KUBEBUILDER_ASSETS:="${HOME}/.kubebuilder/bin"}"
+KUBEBUILDER_ASSETS="${KUBEBUILDER_ASSETS:="/usr/local/kubebuilder/bin"}"
```

Comment:
Do you even need to set this, if it's the internal default for kubebuilder?

Reply:
We do a make directory and a find call in this script, so I think this _does_ need to be set. I could go either way on whether the variable should be statically defined or not though. Thoughts?